src/util.ts

```diff
@@ -90,6 +90,44 @@ export async function showWorkspaceFoldersQuickPick(placeHolderString: string, t
     }
 }
 
+export async function showWarQuickPick(placeHolderString: string, telemetryProperties: TelemetryProperties): Promise<string> {
+    const warFiles: vscode.Uri[] = await vscode.workspace.findFiles('**/*.war');
+    const warQuickPickItems: IAzureQuickPickItem<string | undefined>[] = warFiles.map((uri: vscode.Uri) => {
+        return {
+            label: path.basename(uri.fsPath),
+            description: uri.fsPath,
+            data: uri.fsPath
+        };
+    });
+
+    warQuickPickItems.unshift({ label: '$(package) Browse...', description: '', data: undefined });
```

Comment:
I'd prefer if browse was the last option - I think normally user's will just pick the first item from the workspace, right?